MAINTAINERS.md

```diff
@@ -165,20 +256,22 @@ to pass.
   ```
```

Comment:
Commenting here but it's about the section above. I believe publish to NPM should only happen after the publish PR is approved. I thought this was already in the Maintainers file, but it might only be in the run book.